### PR TITLE
Webpack and first visual

### PR DESCRIPTION
**Pull request by bland0**
@jase-d @javsflores
Basic work in. Bulk of the PR is HTML/CSS, not too much to go over.

- Switched back to local module-server
- Basic calendar

### CODE REVIEW

**Review by jase-d — APPROVED** [review]
Great work man looks like you put in some time on those calendars and booking. Hard to say how it looks without actually seeing but for the most part I can figure out what every line is doing. 

The only suggestions I have is to separate the Calendar and Booking into separate components just for good practice, and to modify some of the less descriptive variable names to make more sense, idate and pnum kind of throw me off.

However, kind of just searching for a suggestion, for the most part the var names all make sense. Looking to forward to seeing this come to life, keep pushing.

File: client/index.jsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+const BLUE = 'http://localhost:1020'
+
+const MONTHS = [
+  'January', 'February', 'March', 'April',
+  'May', 'June', 'July', 'August',
+  'September', 'October', 'November', 'December'
+]
+
+const Calendar = function(props) {
```
Comment (jonathandavidlewis) [inline client/index.jsx:13]: I would recommend destructuring components like this. It is very common with react components:
Calendar = function ({year, month, dates})
It allows the reader to understand more about the function in the signature. :)

File: server/src/server.js [hunk 2]
```diff
@@ -1,12 +1,14 @@
...
 const db = require('./db');
 
 const PORT = 1020;
 
 const server = express();
 server.use(express.json());
+server.use('/', express.static('public'));
 
-server.get('/:id', function(Q, S) {
+server.get('/id/:id', function(Q, S) {
```
Comment (jonathandavidlewis) [inline server/src/server.js:11]: there should be a THING that is being retrieved.
/THING/:id
/item/:id
/object/:id

Comment (jonathandavidlewis) [reply server/src/server.js:11]: /book/:id.  ?

**Review by jonathandavidlewis — COMMENTED** [review]
Can you just import axios into your app for now and have webpack build everything? why choose to use a separate file?

Your bunndle.js was committed, but should be gitignored in the dist folder.

**Pull request merged**